sdk/core/core/inc/az_span.h

```diff
@@ -243,6 +243,37 @@ AZ_INLINE void az_span_set(az_span span, uint8_t fill) {
   memset(az_span_ptr(span), fill, az_span_capacity(span));
 }
 
+/**
+ * @brief             Appends one char to the #az_span if there is enough capacity for it.
+ * @param self        #az_span where to append the char.
+ * @param c           Char to be appended.
+ * @param out[out]    #az_span returned as result of the concatenation.
+ * @return            An #az_result value indicating the result of the operation.
+ * @retval            #AZ_OK                      If no errors occurs.
+ * @retval            #AZ_ERROR_BUFFER_OVERFLOW   If there is not enough capacity left to append the `c`.
+ */
+AZ_NODISCARD az_result az_span_append_byte(az_span self, char c, az_span * out);
+
+/**
+ * @brief             Appends one uint32_t to the #az_span if there is enough capacity for it.
+ * @param self        #az_span where to append the value.
+ * @param n           Value to be appended.
+ * @return            An #az_result value indicating the result of the operation.
+ * @retval            #AZ_OK                      If no errors occurs.
+ * @retval            #AZ_ERROR_BUFFER_OVERFLOW   If there is not enough capacity left to append the `n`.
+ */
+AZ_NODISCARD az_result az_span_append_uint32(az_span * self, uint32_t n);
+
+/**
+ * @brief             Appends one int32_t to the #az_span if there is enough capacity for it.
+ * @param self        #az_span where to append the value.
+ * @param n           Value to be appended.
+ * @return            An #az_result value indicating the result of the operation.
+ * @retval            #AZ_OK                      If no errors occurs.
+ * @retval            #AZ_ERROR_BUFFER_OVERFLOW   If there is not enough capacity left to append the `n`.
+ */
+AZ_NODISCARD az_result az_span_append_int32(az_span * self, int32_t n);
```

Comment:
should be:
AZ_NODISCARD az_result az_span_append_i32toa(az_span span, int32_t n, az_span * out_span);

Reply:
Got it. Changed.